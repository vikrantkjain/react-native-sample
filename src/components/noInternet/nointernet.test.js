import renderer from 'react-test-renderer';
import NetProvider from '../../contexts/NetworkProvider';

describe('Test NetProvider', () => {
  const tree = renderer.create(
    <NetProvider>
      <></>
    </NetProvider>,
  );
  it('Render successfully', () => {
    expect(tree).toBeDefined();
  });
  it('Render correctly', () => {
    const treeToJSON = tree.toJSON();
    expect(treeToJSON).toMatchSnapshot();
  });
});
