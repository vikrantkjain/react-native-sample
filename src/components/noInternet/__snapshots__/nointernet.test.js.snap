// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NetProvider Render correctly 1`] = `null`;
